server/server.go

```diff
@@ -215,6 +216,11 @@ func (s *Server) Init(ctx context.Context) (*Server, error) {
 	}
 	s.DiagnosticHandler = s.initHandlerAuthn(s.DiagnosticHandler)
 
+	s.Handler, err = s.initHandlerDecodingLimits(s.Handler)
```

Comment:
Isn't this backwards incompatible? I mean even with the `nil` config we set the defaults, so would this not break existing behavior? Now if we're treating this as an attack surface for OPA, it makes sense to fix it. So having a higher default will probably not break most users.

Reply:
> This is a potentially backwards-incompatible change

We should add a note in the Changelog about this and also an explanation behind the justification for this change would be helpful.